### PR TITLE
Practice PR: add feedback when toggling snap in main section, matches MIDI editor feedback

### PR DESCRIPTION
**Pull request by jennykbrennan**

### CODE REVIEW

File: src/reaper_osara.cpp [hunk 1]
```diff
@@ -2282,6 +2282,8 @@ map<pair<int, int>, ToggleCommandMessage> TOGGLE_COMMAND_MESSAGES = {
 	{{MAIN_SECTION, 42363}, {_t("samples secondary"), nullptr}}, // View: Secondary time unit for ruler: Samples
 	{{MAIN_SECTION, 42362}, {_t("seconds secondary"), nullptr}}, // View: Secondary time unit for ruler: Seconds
 	{{MAIN_SECTION, 14}, {_t("master muted"), _t("master unmuted")}}, // Track: Toggle mute for master track
+	{{MAIN_SECTION, 1157}, {_t("enabled snap"), _t("disabled snap")}}, // Options: Toggle snapping
+
```
Comment (jcsteh) [inline src/reaper_osara.cpp:2286]: nit: Remove extraneous blank line.

```suggestion
```

**Review by jcsteh — APPROVED** [review]
Nice! Thanks. Just one tiny nit that I'm going to just fix while I'm here.

**Comment by ptorpey** [comment]
Just FYI, when I tried to install in Windows, the installer threw an error saying that it couldn’t write the DLL file and asking if I wanted to retry or abort or ignore.  I’ve never seen this error with an Osara build.

 

--Pete

**Comment by RDMurray** [comment]
@ptorpey  Is Reaper still running? This is the error you get if you try to install OSARA without closing Reaper first.

**Comment by ptorpey** [comment]
Ah, my bad.  Reaper was open at the time.  After shutting down Reaper the install went flawlessly.

 

Thanks.

 

--Pete

**Pull request merged**